### ISSUE TITLE
'LineString' object has no attribute 'geoms'

### ISSUE THREAD

**Issue by tyrosinase** [inkstitch/inkstitch#2665]
My apprentice (the college kid who lives with us) has been playing around with Ink/Stitch and has discovered a bug (at least on my bleeding-edge install). The skully shapes have given me transient issues before, but they're being particularly hard-to-nail-down this time. (Don't mind the layer choices, he's still learning. Also he was perhaps optimistic on the filename...)

Ink/Stitch Version: initial (linux)
```

Traceback (most recent call last):
  File "lib/elements/element.py", line 603, in handle_unexpected_exceptions
  File "lib/elements/element.py", line 571, in embroider
  File "lib/elements/fill_stitch.py", line 762, in to_stitch_groups
  File "lib/elements/fill_stitch.py", line 814, in do_underlay
  File "lib/debug.py", line 231, in decorated
  File "lib/stitches/auto_fill.py", line 90, in auto_fill
  File "lib/debug.py", line 231, in decorated
  File "lib/stitches/auto_fill.py", line 690, in path_to_stitches
  File "lib/stitches/auto_fill.py", line 650, in travel
  File "lib/utils/clamp_path.py", line 120, in clamp_path_to_polygon
AttributeError: 'LineString' object has no attribute 'geoms'
```

![emojihearts_done](https://github.com/inkstitch/inkstitch/assets/9473609/b5f3d3d3-180e-4bac-a7a6-06ad4cb81fec)

**Comment by kaalleen** [comment]
Nice find thanks. Problems seems to be duplicated points like this one in path73270-7
![Bildschirmfoto vom 2023-12-31 11-05-45](https://github.com/inkstitch/inkstitch/assets/36401965/af578727-8d64-43ef-a320-dc38b74f0b17)

**Comment by tyrosinase** [comment]
Ah, that'll be it then. Troubleshooting said it was a border crossing itself, last thing before I went to bed, and I looked at the bezier handles and didn't see anything and decided to tackle it in the morning.

(Skully is a Twemoji, and the paths always have weird artifacts from whatever high-level software they use to generate them. People look down on Inkscape but dangit, you don't get that weirdness when you're down among the nodes yourself.)
